lib/screens/auth/login.dart

```diff
@@ -131,9 +131,11 @@ class _LoginScreenState extends State<LoginScreen> {
       // successfully verified phone number
       var bdy = json.decode(response.body);
       String token = bdy["token"];
+      String far = bdy["far"];
```

Comment:
in the `bdy["far"]`, that data type returned is a `boolean`. Is it optimal to store as a string ? Or does dart cast it automatically from `boolean` to `String` type ?